lib/shovel.js

```diff
@@ -117,8 +106,7 @@ function setupFiles(opts, strategies) {
     if (opts.files) {
         if (strategies.files) {
             strategies.files();
-        }
-        else {
+        } else {
```

Comment:
Sorry about that. I thought the majority of the code had `} else {` and misunderstood your comment about `{` falling to the next line.